build.properties

```diff
@@ -10,10 +9,12 @@ mod_url=https://github.com/Team-RTG/Realistic-Terrain-Generation
 mod_author="Team RTG"
 mod_creds=Forked from RWG by ted80. OpenSimplex Noise by KdotJPG.
 mod_logo=assets/rtg/logo.png
-mcf_version=12.18.1.2011
-# mcf_postfix is the branch postfix (without '-') on the Forge version when it's not the default branch in the Forge repo
-# This should be same as mc_version, eg Non-default: 1.9.4-12.17.0.1974-1.9.4, Default: 1.9.4-12.17.0.1976
-mcf_postfix=
-mappings=snapshot_nodoc_20160724
+mcf_version=12.18.1.2094
```

Comment:
I noticed quite a few commits since 2011 that *could* affect RTG:

* [Fix async race condition on new chunks](https://github.com/MinecraftForge/MinecraftForge/commit/cc0e6a182568a282c1daa61ad300c0d29d422f45)
* [Update decorate for biomes to support Decorate event](https://github.com/MinecraftForge/MinecraftForge/commit/0cf2cdfdb3fa603379d165c2e46cb0b57383501d)
* [Fix a vanilla bug with the blockstate ID map not being properly symmetric with respect to block.getStateFromMeta](https://github.com/MinecraftForge/MinecraftForge/commit/ab64d1a1665e93b3d1fa46065119356eabe7efd3)
* [Fix vanilla bug in BlockFalling that caused blocks to loose state info during world gen](https://github.com/MinecraftForge/MinecraftForge/commit/b5db198f8b97618febd7a48a57cd4f64fdd23e25)

Whilst most of those *should* be changes for the better, we don't know for certain that they won't break something in RTG, or introduce some weird incompatibility involving RTG and some other mod.

I also really, *really* don't like building against a non-recommended version of Forge. It's recommended for a reason - it's supposed to serve as a common playing field for developers and users alike. By building against a non-recommended version of Forge, we're basically throwing all of that out the window.

I don't know... am I making mountains out of molehills here? Every instinct I have is telling me that this is a bad idea when we're 3 days from release, but... does it really matter? What's the worst that could happen... people start posting tons of crash reports due to some obscure incompatibility with the latest Forge, so we revert back to the recommended version of Forge and release 4.1.1.3 the next day?